### PR TITLE
Add book styling and a object class

### PR DESCRIPTION
**Pull request by idungstanley**
* We use javascript classes instead of the objects and arrays
* We create class methods to add and remove books
* We made use of CSS styles to match the wireframe.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,69 +1,80 @@
...
+// Creating a book class
+class Book {
+  constructor(title, author, id = new Date().getTime()) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
+}
+
 let array = [];
```
Comment (SajjadAhmad14) [inline main.js]: The array should be declared as an instance variable of `Book` not outside of class.

File: main.js [hunk 2]
```diff
@@ -1,69 +1,80 @@
...
+    author.value = '';
+  }
 }
 
-function getStorage() {
-  if (localStorage.getItem('books') === null) {
-    setStorage();
-  } else {
+// Store data in local storage
+class bookStore {
```
Comment (SajjadAhmad14) [inline main.js]: There is no need of declaring multiple classes. You can declare methods for adding, removing, and rendering book items in the `Book` class.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi team, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [x] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Emmyn5600 — APPROVED** [review]
Hi @idungstanley and @Amedzro-Elikplim 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada: :clap: :clap: :100:

nodding-yes

## Optional suggestions

- [x] N/A

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**